src/com/jedk1/jedcore/JCManager.java

```diff
@@ -6,9 +6,8 @@
  import com.jedk1.jedcore.ability.waterbending.HealingWaters;
  import com.jedk1.jedcore.ability.waterbending.IcePassive;
  import com.jedk1.jedcore.util.RegenTempBlock;
- import com.jedk1.jedcore.util.TempFallingBlock;
 
-public class JCManager implements Runnable {
+ public class JCManager implements Runnable {
```

Comment:
AHHHHHHHHH

Reply:
I DONT KNOW HOW THIS HAPPENED, ILL FIX IT